web/html/src/components/coco-attestation/CoCoResult.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from "react";
+
+import { BootstrapPanel } from "components/panels/BootstrapPanel";
+
+import { AttestationResult, renderTime } from "./Utils";
+
+type Props = {
+  /** The id of the system to be shown */
+  result: AttestationResult;
+};
+
+type State = {};
```

Comment:
Since the state is always empty, we can drop it altogether. That way the constructor becomes redundant and we can drop that, too.

```tsx
class CoCoResult extends React.Component<Props> {
  render() ...
}
```

Reply:
Thanks, I didn't know it was possibile. I removed the unneeded state.